resources/cdk/photo_asset_manager/SPECIFICATION.md

```diff
@@ -140,23 +140,23 @@ Each language will implement these Lambda functions. These functions handle the
 
 ### ⭐ Upload
 
-1. Create a presigned Amazon S3 URL in the Storage Bucket. The object key shall be the provided file name, prefixed with a UUIDv4 and a /. The presigned URL will allow a `PUT` method operation with `Content-Type` `image/jpeg`. The expiration will live for 5 minutes (5 \* 60 \* 1000 ms = 300,000 ms).
+1. Create a presigned Amazon S3 URL in the Storage Bucket. The object key shall be the provided file name, prefixed with a UUIDv4 and a `/`. The presigned URL will allow a `PUT` method operation with `Content-Type` `image/jpeg`. The expiration will live for 5 minutes (5 \* 60 \* 1000 ms = 300,000 ms).
 2. Return a serialized JSON object with a single key, “url”, containing the value of the presigned URL.
 
 ### ⭐ DetectLabels
 
 This Lambda will be triggered by uploads to the Storage Bucket.
 
 1. Run Amazon Rekognition’s `detectLabels` on each incoming object.
-2. If Amazon Rekognition’s `detectLabels` succeeds, update the Labels Table. For each Label key, add the image object key to the list in the Images column and increment the Count column. These must be atomic operations. If the enhanced DynamoDB client supports atomic counter fields, use them. Otherwise, the request can use update expressions to atomically update Count and Images.
+2. If Amazon Rekognition’s `detectLabels` succeeds, update `LabelsTable`. For each Label key, add the image object key to the list in the Images column and increment the Count column. These must be atomic operations. If the enhanced DynamoDB client supports atomic counter fields, use them. Otherwise, the request can use update expressions to atomically update Count and Images.
```

Comment:
This was fine as just Labels table, because the table would be called `{NAME}-PAM-PamTablesLabelsTable{RANDOM}` not `LabelsTable`